app/assets/javascripts/discourse/app/lib/plugin-api/value-transformer.js

```diff
@@ -0,0 +1,137 @@
+// add core transformer names
+const validCoreTransformerNames = new Set(["header-notifications-avatar-size"]);
+
+// do not add anything directly to this set, use addTransformerName instead
+const validPluginTransformerNames = new Set();
+
+const transformersRegistry = new Map();
```

Comment:
We'll need a way to reset these in [testCleanup](https://github.com/discourse/discourse/blob/0b10e335aedb06cb8a1b09549076175546f55b75/app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js#L175C24-L175C24)

Reply:
the functions are there, just at the bottom of the file (and unused yet)

Reply:
I wasn't aware that `testCleanup` existed. 😄

Reply:
Reviewed the function and added a cal in `testCleanup`